tutorials/run-spring-petclinic-on-app-engine-flex-cloudsql.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Run Spring Pet Clinic with CloudSQL on Google App Engine Flexible Environment
+description: Learn how to deploy Spring Boot Pet Clinic application to Google App Engine flexible environment and use CloudSQL.
+author: jishaa
+tags: App Engine Flex, CloudSQL, Spring Boot
+date_published: 2017-05-12
+---
+
+This tutorial will walk you through getting Spring Pet Clinic application up and running on App Engine Flex with CloudSQL.
+
+### Spring Boot and Spring Pet Clinic
+[Spring Boot](https://projects.spring.io/spring-boot/) provides Java developers a quick annotation driven way to deploy services using minimal code.
+[Spring Pet Clinic](https://github.com/spring-projects/spring-petclinic) is a popular Spring application that will quickly demonstrate the power of SpringBoot.
+
+### App Engine Flex environment for Java
+[App Engine Flex](https://cloud.google.com/appengine/docs/flexible/java/) provide the ability to run your Java applications in Docker containers on GCE machines, with the ability
+to have them deployed, monitored and auto-scaled.
+We will be using the [Java 8 Runtime environment](https://cloud.google.com/appengine/docs/flexible/java/dev-java-only) for this tutorial.
+
+### CloudSQL
+[CloudSQL for MySQL](https://cloud.google.com/sql/docs/mysql/) is a fully managed MySQL database service on GCP.
+
+### Prerequisites
+1. Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com/).
+1. Enable billing for your project.
+1. Install the [Google Cloud SDK](https://cloud.google.com/sdk/).
+
+### Prepare
+1. Initialize, create an app engine application and authorize `gcloud` for using GCP APIs in your local environment :
```

Comment:
replace with "Initialize the Cloud SDK, create an App Engine application, and authorize the Cloud SDK for using GCP APIs in your local environment:"

Also, global: No space before the colon at the end of the line.